opflexagent/gbp_ovs_agent.py

```diff
@@ -112,6 +123,20 @@ def __init__(self, root_helper=None, **kwargs):
         # TODO(ivar): make this component pluggable.
         self.ep_manager = ep_manager.EndpointFileManager().initialize(
             self.host, self.bridge_manager, kwargs)
+        self.init_firewall()
+
+    def _invoke_ep_mgr(self, port):
```

Comment:
Just thinking about your TODO in the code -- do we anticipate other types of EP managers? If so, then I guess we are already in the world of subclassing, so it makes more sense to do it directly here (although then you can have misconfigurations -- DVS w/wrong EP manager... but there's the other pitfall of tying it to hypervisor type, but then not allowing someone to override the EP manager if they want).